src/index.js

```diff
@@ -110,6 +110,19 @@ export default function onClickOutsideHOC(WrappedComponent, config) {
       this.disableOnClickOutside();
     }
 
+    /**
+     * Options for addEventHandler and removeEventHandler
+     */
+    getEventHandlerOptions(eventName) {
+      let handlerOptions = null;
```

Comment:
Could you just hoist this to the top of the file and accept `instance` as argument? there is no point to keep this on prototype and expose this as "public" API

Reply:
yup! done.